### ISSUE TITLE
[Switch] Download count on packages is always zero

### ISSUE THREAD

**Issue by clamintus** [fortheusers/hb-appstore#64]
This has been going for some months: before that I used to see non-zero download counts when I selected a package I wanted to download. But now for every homebrew app, the download count shown is zero

**Comment by compucat** [comment]
Good eye! This is the result of a couple things going on behind the scenes - we'd taken down the old statistics system while working on a new one, but progress seems to have stalled on that. @vgmoose might have more relevant info

**Comment by clamintus** [comment]
I see... I suspected something like this, not that I'm a fully knowledged programmer like you (I don't even completely know C++ :p) but I can understand its logic. Thanks for considering fixing the bug!

**Comment by vgmoose** [comment]
Yeah, we've really let this issue go on. As we scaled and time went on, our old stats system stopped being usable. We then set out to overhaul it, (With daily graphs, realtime stats, and support for our new build system) but didn't get fully there! Then, like 6 months passed. 🙂 

Fortunately, LyfeOnEdge and pwsincd were able to get a workaround working for the old method that does scale, but we never got around to going the final mile and hooking it up and getting it to run automatically.

We've just now ran it manually and updated the stats. These should be current as of September 10, 2020 (Even though we weren't displaying info for the last several months, the download counts should still be accurate). We'll keep this value updated with the workaround in the near future, and in the hopefully not-to-far future we can get those daily graphs and other enhancements.

Thanks for the issue, the push is definitely appreciated, and it's unfortunate that we had 0 download counts for so long. Somehow we got all the way to September really quickly!

Will reopen if we run into issues again!
